samples/msal-browser-samples/BrokerTestApp2.0/app/brokeredWidget/script.js

```diff
@@ -49,7 +49,77 @@ function handleResponse(resp) {
         // need to call getAccount here?
         const currentAccounts = myMSALObj.getAllAccounts();
         if (!currentAccounts || currentAccounts.length < 1) {
-            return;
+            setTimeout(() => {
+                const contentElement = document.getElementsByClassName("myContent")[0];
+                contentElement.innerHTML = "I am now trying to fetch profile data";
+        
+                setTimeout(async () => {
+                    let exit = false;
+                    const loginReq = { 
+                        scopes: ["openid", "profile", "User.Read"],
+                        loginHint: "idlab@msidlab4.onmicrosoft.com" 
+                    };
+                    // await myMSALObj.ssoSilent(loginReq).then(() => {
```

Comment:
Why is this commented out?

Reply:
Fixed in #2806 but I can uncomment here also. Was done for testing.